src/Controls/src/BindingSourceGen/BindingCodeWriter.cs

```diff
@@ -1,10 +1,15 @@
 using System.CodeDom.Compiler;
 using System.Globalization;
+using System.Runtime.InteropServices;
+
+using static Microsoft.Maui.Controls.BindingSourceGen.UnsafeAccessorsMethodName;
 
 namespace Microsoft.Maui.Controls.BindingSourceGen;
 
 public static class BindingCodeWriter
 {
+	private static readonly string NewLine = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "\r\n" : "\n";
```

Comment:
Could we use Enviornment.NewLine here instead? https://learn.microsoft.com/en-us/dotnet/api/system.environment.newline?view=net-8.0

Reply:
The symbol 'Environment' is banned for use by analyzers: Analyzers should not read their settings directly from environment variables(RS1035)